app/models/view_model/auction.rb

```diff
@@ -103,25 +95,23 @@ def auction_type
     end
 
     def index_bid_summary_partial
-      if single_bid?
-        'auctions/single_bid/index_bid_summary'
-      elsif multi_bid?
-        'auctions/multi_bid/index_bid_summary'
-      end
+      auction.partial_path('index_bid_summary')
     end
 
     def highlighted_bid_info_partial
-      if single_bid?
-        'bids/single_bid/highlighted_bid_info'
-      elsif multi_bid?
-        'bids/multi_bid/highlighted_bid_info'
-      end
+      auction.partial_path('highlighted_bid_info', 'bids')
     end
 
-    delegate :status, :label_class, :label, :tag_data_value_status,
-             :tag_data_label_2, :tag_data_value_2,
-             to: :status_presenter
-
+    delegate(
```

Comment:
could we throw these to the top of the file while we're here? I usually expect delegated methods to be near the top of ruby classes (not sure if this makes me Normal or not)